src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -139,16 +128,44 @@ public double rightVelocity()
     return r_encoder.getVelocity();
 }
 
+public double getLeftEncPosition()
+{
+  return l_encoder.getPosition();
+}
+
+public double getRightEncPosition()
+{
+  return r_encoder.getPosition();
+}
+
 public double fpsToRPM(double fps){
     fps = fps * 12;
     fps = fps/Constants.kWheelCircumference;
     fps = fps *60;
     fps = fps*Constants.kGearRatio;
     return fps;
 }
+
+public void intializeDriveSubystem(CANSparkMax master, CANSparkMax... slaves){
+  master.restoreFactoryDefaults();
+  master.enableVoltageCompensation(12);
+  master.setIdleMode(IdleMode.kBrake);
+  master.setOpenLoopRampRate(.2);
+  //master.setClosedLoopRampRate(0.2);
+ 
+  for(CANSparkMax slave : slaves) {
+    //slave.setClosedLoopRampRate(0.2);
+    slave.restoreFactoryDefaults();
+    slave.enableVoltageCompensation(12);
+    slave.setIdleMode(IdleMode.kBrake);
+    slave.setOpenLoopRampRate(.2);
+    slave.follow(master);
```

Comment:
We need to set smartCurrentLimit. use .setSmartCurrentLimit(40); on each slave too.